app/SearchApi/BcGov.Fams3.SearchApi.Contracts/Person/Address.cs

```diff
@@ -1,12 +1,17 @@
 ﻿using System;
+using System.Collections.Generic;
 using System.ComponentModel;
 
 namespace BcGov.Fams3.SearchApi.Contracts.Person
 {
-    public interface Address
+    public interface Address : BaseMetaData
     {
         [Description("The type of address")]
-        string Type { get; }
+        AddressType Type { get; }
+
+        [Description("The type code of the address directly from data provider")]
+        string TypeCode { get; }
```

Comment:
no TypeCode for searchApi Contract.
Dynadapter will mapping Type to SSG_Address AddressType and SupplierTypeCode.